### ISSUE TITLE
Menu is behind other components, not responding to z-order style

### ISSUE THREAD

**Issue by JamesBender** [reactjs/react-autocomplete#282]
Hi,

I'm using the Autocomplete box in an application along with react-bootstrap Rows and Columns for layout. As shown in the picture, the autocomplete menu is hidden behind some of the other page element. I tried using the menuStyle property to pass in a style with the z-index set, but the component doesn't seem to be picking it up.

Thanks,
James

![image](https://user-images.githubusercontent.com/49018/31356300-93113656-ad0b-11e7-8990-df02ace3e3bb.png)

**Comment by CMTegner** [comment]
Hi James!

Could you provide a reduced test case, or at least your `Autocomplete` configuration (props)? Layout-related problems like these are difficult to debug without enough context, but my gut tells me it's probably due to the ordering and `z-index` values on other elements interfering with each other, and not necessarily a bug in `Autocomplete`.

**Comment by JamesBender** [comment]
Hi Christian,

I've attached the render method with the JSX where I'm using the Autocomplete. It's contained in Well component (we wrote) and is being hidden behind _some_ of the rows in the FormList component (which we also wrote). The Row and Col components are from react-bootstrap (https://react-bootstrap.github.io/) and I'm using the Row component from that library in the FormList as well. 

Thanks,
James


[View.txt](https://github.com/reactjs/react-autocomplete/files/1372098/View.txt)
